### PR TITLE
Updated GenericSchemeMultiCall revert on failed calls

### PR DESCRIPTION
**Pull request by nicoelzer**
- Updated GenericSchemeMultiCall revert on failed calls
- Updated GenericSchemeMultiCall Bool Literals
- Updated GenericSchmeMultiCall events

### CODE REVIEW

File: contracts/schemes/GenericSchemeMultiCall.sol [hunk 1]
```diff
@@ -152,6 +152,8 @@ contract GenericSchemeMultiCall is VotingMachineCallbacks, ProposalExecuteInterf
             } else {
                 (success, genericCallReturnValue) =
                 controller.genericCall(proposal.contractsToCall[i], callData, avatar, proposal.values[i]);
+                /* Whole transaction will be reverted if at least one call fails*/
+                require(success, "Proposal call failed");
```
Comment (orenyodfat) [inline contracts/schemes/GenericSchemeMultiCall.sol:156]: please move that out of the` }` so it will include the `externalTokenApproval` as well .

Comment (nicoelzer) [reply contracts/schemes/GenericSchemeMultiCall.sol]: @orenyodfat just updated.

File: contracts/schemes/GenericSchemeMultiCall.sol [hunk 2]
```diff
@@ -153,6 +153,9 @@ contract GenericSchemeMultiCall is VotingMachineCallbacks, ProposalExecuteInterf
                 (success, genericCallReturnValue) =
                 controller.genericCall(proposal.contractsToCall[i], callData, avatar, proposal.values[i]);
             }
+
+            /* Whole transaction will be reverted if at least one call fails*/
+            require(success, "Proposal call failed");
```
Comment (orenyodfat) [inline contracts/schemes/GenericSchemeMultiCall.sol:158]: please remove the automatic call to `execute`  at https://github.com/daostack/arc/blob/b80fe92ff7dd9eb069c7770431c5dfb041d3e9e1/contracts/schemes/GenericSchemeMultiCall.sol#L116 .
this is not good due to :
1. last voter(executer) does not need to pay for whole execution gas.
2. if execution is reverted in one of the calls.. so the dao cannot decide (for the case of absolute majority)..
 - there suppose to be a differentiation between the dao decision and execution.

**Review by orenyodfat — APPROVED** [review]

**Review by ben-kaufman — APPROVED** [review]
Looks good to me

**Pull request merged**